addon/helpers/pikaday.js

```diff
@@ -11,57 +10,61 @@ deprecate(
 );
 
 const openDatepicker = function(element) {
-  $(element).click();
+  document.querySelector(element).click();
```

Comment:
` document.querySelector(element)` can return null. So we probably want to ensure this is a DOM element before trying to invoke `click()`.